db/schema.rb

```diff
@@ -0,0 +1,201 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20181018082155) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "authentications", force: :cascade do |t|
```

Comment:
Does this create the ids as `bigserial`?

Reply:
Yes,
```
topological_inventory_development=# \d+ authentications
                                                         Table "public.authentications"
     Column     |       Type        | Collation | Nullable |                   Default                   | Storage  | Stats target | Description 
----------------+-------------------+-----------+----------+---------------------------------------------+----------+--------------+-------------
 id             | bigint            |           | not null | nextval('authentications_id_seq'::regclass) | plain    |              | 
```

Looks like in 5.1 `:bigserial` is the default primary key, and in schema_dumper they don't skip the `:id` if it is default: https://github.com/rails/rails/blob/v5.1.6/activerecord/lib/active_record/connection_adapters/abstract/schema_dumper.rb#L13-L14 and https://github.com/rails/rails/blob/v5.1.6/activerecord/lib/active_record/connection_adapters/postgresql/schema_dumper.rb#L19-L21